bin/getopt.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/false
+# This file should be source-d rather than exec-ed
+
+coloredtest=0
+while : ; do
+    case "$1" in
+        --evaluatesh=*|--compilesh=*)
+            case "$1" in
+                --evaluatesh=debug|--compilesh=debug)
+                    export RUST_BACKTRACE=1
+                    debugme=1
```

Comment:
Oh and the backtrace thing would be rust-lang/rust-playpen#119, so it should probably be solved in a different way (such as a check box on the website - but that can wait for another PR, too)